content/en/docs/installation/supported-releases.md

```diff
@@ -19,14 +19,14 @@ release every two months.
 
 | Release | Release Date |  End of Life | [Supported Kubernetes versions][s] | [Supported OpenShift versions][s] |
 |---------|:------------:|:------------:|:----------------------------------:|:---------------------------------:|
-| [1.7][] | Jan 26, 2022 | May 26, 2022 |             1.18 → 1.23            |             4.5 → 4.9             |
-| [1.6][] | Oct 26, 2021 | Mar 30, 2022 |             1.17 → 1.22            |             4.4 → 4.9             |
+| [1.7][] | Jan 26, 2022 | May 26, 2022 |             1.18 → 1.23            |             4.5 → 4.10            |
```

Comment:
note: updated the openshift version to to match what I think was always the case, since 4.10 == k8s 1.23